### ISSUE TITLE
[FEATURE]: Can gradient accumulation be used in the pretraining of llama2-70b?

### ISSUE THREAD

**Issue by InvalidFreedom** [hpcaitech/ColossalAI#4785]
### Describe the feature

Can gradient accumulation be used in the pretraining of llama2-70b?
And if so, how can it be enabled？

**Comment by Fridge003** [comment]
您好，gemini对梯度累积的支持已经完成。
Hello, we have just supported gradient accumulation on Gemini plugin.
Usage can be referred to  `docs/source/en/features/gradient_accumulation_with_booster.md`, our online tutorials will also be updated within a few days.
